ui/src/app/components/breadcrumb.tsx

```diff
@@ -1,20 +1,27 @@
-import {Component} from '@angular/core';
 import * as fp from 'lodash/fp';
 import * as React from 'react';
 
 import {dropNotebookFileSuffix} from 'app/pages/analysis/util';
 import colors from 'app/styles/colors';
 import {
-  ReactWrapperBase,
   withCurrentCohort,
   withCurrentConceptSet,
   withCurrentWorkspace,
   withRouteConfigData,
   withUrlParams
 } from 'app/utils';
 import {BreadcrumbType, navigateAndPreventDefaultIfNoKeysPressed} from 'app/utils/navigation';
+import {WorkspaceData} from 'app/utils/workspace-data';
+import {Cohort, ConceptSet} from 'generated/fetch';
 
 const styles = {
+  breadcrumb: {
+    marginLeft: '3.25rem',
+    display: 'inline-block',
+  },
+  primary: {
+    color: colors.primary
```

Comment:
not sure if this needs to be extracted since its just 1 property and that too call to colors class, normally we extract styles having more than 3 properties

Reply:
But since its variable in colors file, its already adding anything